src/site/content/en/blog/min-max-clamp/index.md

```diff
@@ -0,0 +1,179 @@
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/min-max-clamp/min-width.mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Using the clamp() function to limit a minimum and maximum width.
+  </figcaption>
+</figure>
+
+## Padding Management
+
+Using the same concept as above, where the `min()` function can set a “max” value and `max()` sets a “min” value, you can use `max()` to set a minimum padding size. This example comes from [CSS Tricks](https://css-tricks.com/using-max-for-an-inner-element-max-width/), where reader Caluã de Lacerda Pataca shared this idea:
+The idea is to enable an element to have additional padding at larger screen sizes, but maintain a minimum padding at smaller screen sizes, particularly on the inline padding. To achieve this, use `calc()` and subtract the minimum padding from either side: `calc((100vw - var(--contentWidth)) / 2)`, *or* use max: `max(2rem, 50vw - var(--contentWidth) / 2)`. All together it looks like:
+
+```css
+footer {
+  padding: var(--blockPadding) max(2rem, 50vw - var(--contentWidth) / 2);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/min-max-clamp/min-padding.mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Setting a minimum padding for a component using the max() function. <a href="https://codepen.io/chriscoyier/pen/qBZqNKa">See Demo on Codepen.</a>
+  </figcaption>
+</figure>
+
+## Fluid Typography
+
+In order to enable [fluid typography](https://www.smashingmagazine.com/2016/05/fluid-typography/), [Mike Riethmeuller](https://twitter.com/mikeriethmuller) popularized a technique that uses the `calc()` function to set a minimum font size, maximum font size, and allow for scaling from the min to the max.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/min-max-clamp/fliud-type.mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Creating fluid typography with clamp(). <a href="https://codepen.io/una/pen/ExyYXaN">See Demo on Codepen.</a>
+  </figcaption>
+</figure>
+
+With `clamp()`, you can write this more clearly. Rather than requiring a complex string, the browser can do the work for you. Set the minimum acceptable font size (for example, `1.5rem` for a title, maximum size (i.e. `3rem`) and ideal size of `5vw`. 
+
+Now, we get typography that scales with the viewport width of the page until it reaches the limiting minimum and maximum values, in a much more succinct line of code: 
+
+```css
+p {
+  clamp(1.5rem, 5vw, 3rem);
+}
+```
+
+
+## Conclusion
+The CSS math functions, `min()`, `max()`, and `clamp()` are very powerful, relatively well supported, and could be just what you're looking for to help you build responsive UIs. For more resources, check out:
```

Comment:
Relatively well supported? In the intro we said they're supported in all browsers. Can we update this (or the intro) to make sure our messaging is consistent?